string label
float64 probability
BoundingBox boundingBox